<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminInfoForm_Chat</class>
 <widget class="QWidget" name="AdminInfoForm_Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>메시지 입력</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="chatDisplay">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>781</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>480</y>
     <width>196</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_fileAdmin">
      <property name="text">
       <string>파일 첨부 하기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_admin">
      <property name="text">
       <string>전송하기</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>641</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
